Dockerfile

```diff
@@ -19,15 +19,45 @@ MAINTAINER Jason C. Kwan "jason.kwan@wisc.edu"
 # You should have received a copy of the GNU Affero General Public License
 # along with Autometa. If not, see <http://www.gnu.org/licenses/>.
 
-RUN conda config --prepend channels erees \
-    && conda config --append channels bioconda \
+RUN conda config --append channels bioconda \
     && conda config --append channels conda-forge
 
-RUN conda install autometa tsne
+RUN conda install \
+    biopython \
+    pandas \
+    tqdm \
+    tsne \
+    numpy \
+    scikit-learn \
+    scikit-bio \
+    samtools \
+    bedtools \
+    bowtie2 \
+    hmmer \
+    prodigal \
+    diamond \
+    nextflow \
+    parallel \
+    requests \
+    umap-learn \
+    hdbscan -y
 
-RUN echo "testing autometa and tsne import"
-RUN python -c "import autometa"
-RUN python -c "import tsne"
+RUN git clone https://github.com/KwanLab/Autometa.git \
+    && cd Autometa \
+    && git checkout dev \
+    && python setup.py install
 
-RUN echo "Running Autometa dependencies test"
-RUN autometa --check-dependencies --debug
+RUN echo "testing autometa import" \
+    && python -c "import autometa"
+
+# Check entrypoints are available
+RUN autometa-length-filter -h \
+    & autometa-orfs -h \
+    & autometa-coverage -h \
+    & autometa-kmers -h \
+    & autometa-markers -h \
+    & autometa-taxonomy -h \
+    & autometa-taxonomy-lca -h \
+    & autometa-taxonomy-majority-vote -h \
+    & autometa-binning -h \
+    & autometa-unclustered-recruitment -h
```

Comment:
The output of this only shows up when you build the docker image, not when you run it. So no one will see this. Perhaps put this in the bash rc file?

Reply:
Do you want the help text for all of the entrypoints to be printed to the user each time a container is spawned from the image? This is a minimal image as it does not have any of the required databases configured. The user will need to mount their databases to the docker container and then have to point autometa to these mounted paths. That being said, I think nextflow can be configured such that the user will not have to worry about issuing these docker commands. We can just hardcode the docker image into each process and nextflow will do the rest.

Reply:
Oh, I see. It looked like your intention was to show the messages to the user. In that case, you could probably take them out after showing it works. On another note though, if the Docker is intended to be used interactively, then a message upon startup might be a good idea.